### DISCUSSION TITLE
Grove - IMU 9DOF(lcm20600+AK09918) | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1917] [Q&A]
# Grove - IMU 9DOF(lcm20600+AK09918) | Seeed Studio Wiki

Grove - IMU 9DOF(lcm20600+AK09918)

https://wiki.seeedstudio.com/Grove-IMU_9DOF-lcm20600+AK09918/

**Comment by jcs-consult** [comment]
This is an informative tutorial... unfortunately it fails to mention that the library does not work as downloaded.  The IC2dev files need to be replaced/upgraded manually to reflect their current status before the examples will compile. Please update the library to reflect this.
